src/widgets/tables/data_preview_screen.py

```diff
@@ -1,41 +1,112 @@
+                self.render_data(self.estimation_header_click, self.estimation_cell_click)
+            case PreviewReason.REDUCTION:
+                self.render_instruction("Double click on a header to change column name")
+                self.render_data(self.reduction_header_click, fallback)
+            case PreviewReason.PREVIEW:
+                self.render_data(fallback, fallback)
+
         self.setLayout(self.layout)
 
     def render_instruction(self, instruction):
         instruction_widget = QLabel(instruction)
         self.layout.addWidget(instruction_widget)
 
-    def render_data(self):
+    def render_data(self, handleHeaderClick, handleCellClick):
         self.data_table.setSizePolicy(QSizePolicy.Expanding, QSizePolicy.Expanding)
         self.data_table.setModel(QtTable(self.engine.state.imported_data))
-        self.data_table.horizontalHeader().sectionDoubleClicked.connect(self.handleHeaderClick)
+        self.data_table.horizontalHeader().sectionDoubleClicked.connect(handleHeaderClick)
+        self.data_table.doubleClicked.connect(handleCellClick)
         self.layout.addWidget(self.data_table)
 
-    def handleHeaderClick(self, index):
+    def reduction_header_click(self, index):
         newHeader, ok = QInputDialog.getText(self,
-                                                   'Change header label for column %d' % index,
-                                                   'Header:',
-                                                   QLineEdit.Normal,
-                                                   "")
+                                             'Change header label for column %d' % index,
+                                             'Header:',
+                                             QLineEdit.Normal,
+                                             "")
         if ok:
             self.engine.rename_column(index, newHeader)
             self.data_table.setModel(QtTable(self.engine.state.imported_data))
 
+    def estimation_header_click(self, index):
+        newValue, ok = QInputDialog.getText(self,
+                                            'Default value for column %d:' % index,
+                                            'Value:',
+                                            QLineEdit.Normal,
+                                            "")
+        if ok:
+            column = self.engine.state.imported_data.iloc[:, index]
+            newValue = self.cast_input_type(column.dtype, newValue)
+            if newValue is not None:
+                column.fillna(newValue, inplace=True)
+            self.data_table.setModel(QtTable(self.engine.state.imported_data))
+
+    def estimation_cell_click(self):
+        cell = self.data_table.selectionModel().selectedIndexes()[0]
+        row, col = cell.row(), cell.column()
+        newValue, ok = QInputDialog.getText(self,
+                                            'Default value for cell (%d, %d):'.format(row, col),
```

Comment:
```suggestion
                                            f'Default value for cell ({row}, {col}):',
```
the current way doesn't work. `f""` vs `"".format` - I like both, up to you